src/gui/optionsdialog.ui

```diff
@@ -2934,7 +2934,7 @@ Disable encryption: Only connect to peers without protocol encryption</string>
                <item row="0" column="0">
                 <widget class="QCheckBox" name="checkMaxRatio">
                  <property name="text">
-                  <string>When ratio reaches</string>
+                  <string>When ratio reaches:</string>
```

Comment:
I believe it is incorrect to use `:` in this case and similar cases below. Look at the resulting sentence:
>When ratio reaches 1.5 then stop torrent.

Reply:
But it is not a sentence in the UI. It is a label with a spinbox. Adding colon at the end would make the option fields look uniform. This is the only group that doesn't have colon at the end of the label.

To be honest, I don't insist making this change. I'll revert it if others think the same.

Reply:
I have no opinion / both looks fine to me.

Reply:
> I'll revert it if others think the same.

Ok.

Reply:
Reverted.